mkdocs.yml

```
@@ -51,7 +51,13 @@ nav:
     - Services:
       - Core: 
         - microservices/core/Ch-CoreServices.md
-        - microservices/core/data/Ch-CoreData.md
+        - Core Data:
+            - Purpose: microservices/core/data/Purpose.md
+            - Getting Started: microservices/core/data/GettingStarted.md
+            - Configuration: microservices/core/data/Configuration.md
+            - API Reference: microservices/core/data/APIReference.md
+            - Source Code: microservices/core/data/SourceCode.md
+            - Events and Readings: microservices/core/data/EventsAndReadings.md
```

I was expecting this so it is consistent with the app service.
```suggestion
            - Additional Details:
               - Events and Readings: microservices/core/data/details/EventsAndReadings.md
```